account_payment_order_notification/tests/test_account_payment_order_notification.py

```diff
@@ -22,7 +24,7 @@ def setUpClass(cls):
         )
         cls.partner_c = cls.env["res.partner"].create({"name": "Test partner C"})
         cls.product = cls.env["product.product"].create(
-            {"name": "Test product", "list_price": 100}
+            {"name": "Test product", "lst_price": 100}
```

Comment:
Why are you changing the `list_price` field to `lst_price`? (I don't think it's necessary).